core/src/lib.rs

```diff
@@ -7,18 +7,18 @@ extern crate alloc;
 pub mod chiplets;
 pub mod errors;
 
-pub use ::crypto::{Word, EMPTY_WORD, ONE, WORD_SIZE, ZERO};
+pub use ::miden_crypto::{Word, EMPTY_WORD, ONE, WORD_SIZE, ZERO};
```

Comment:
minor nit: we can probably just say:
```
pub use miden_crypto::{Word, EMPTY_WORD, ONE, WORD_SIZE, ZERO};
```
(without the `::` in front of `miden-crypto`).